### PR TITLE
Multiple updates

### PR DESCRIPTION
**Pull request by arthurbarr**
1. Add warning message about using the master branch
2. Fix REST messaging
3. Fix use of image creation date on Mac.

### CODE REVIEW

File: Dockerfile-server [hunk 1]
```diff
@@ -26,7 +26,7 @@ ARG IMAGE_SOURCE="Not specified"
 COPY cmd/ ./cmd
```
Comment (parrobe) [inline Dockerfile-server]: Need to remove the `ARG IMAGE_CREATED="Not specified"` that is 2 lines up as it is no longer needed. Will lower our layers by 1

File: cmd/runmqserver/webserver.go [hunk 2]
```diff
@@ -33,7 +35,22 @@ func startWebServer() error {
 		return nil
 	}
 	log.Println("Starting web server")
-	out, rc, err := command.RunAsMQM("strmqweb")
+	cmd := exec.Command("strmqweb")
+	// Set a default app password for the web server, if one isn't already set
+	_, set := os.LookupEnv("MQ_APP_PASSWORD")
+	log.Println(cmd.Env)
```
Comment (parrobe) [inline cmd/runmqserver/webserver.go]: Should we be printing out the env given that we are storing passwords in the env? Could result in passwords being transmitted to any container log scrapping services.

File: cmd/runmqserver/webserver.go [hunk 3]
```diff
@@ -33,7 +35,22 @@ func startWebServer() error {
...
+	if !set {
+		// Take all current environment variables, and add the app password
+		cmd.Env = append(os.Environ(), "MQ_APP_PASSWORD=passw0rd")
+	}
+	log.Println(cmd.Env)
```
Comment (parrobe) [inline cmd/runmqserver/webserver.go]: Same here.

Comment (arthurbarr) [reply cmd/runmqserver/webserver.go]: That's debug information which I forgot to take out.  Well spotted... the system works!

**Comment by arthurbarr** [comment]
Fixes #178

**Pull request merged**